lib/models/ObjectMD.js

```diff
@@ -105,6 +106,9 @@ class ObjectMD {
             'x-amz-restore': {
                 'ongoing-request': false,
             },
+            'archive': {
```

Comment:
Is this necessary to store on every single new object that's created or updated, even if there's no cold storage capability on the platform (probably the majority of cases). This adds 20-ish bytes per object, multiplied by billions, where metadata space is precious.

Just noticed the same for `x-amz-restore`.

Reply:
I guess you meant "not necessary", and I agree :smile: I removed the fields from the default object MD: https://github.com/scality/Arsenal/pull/1856/files#diff-073d3a5a6a278c5758cbf757c53c2a5e04843f623799fbfd51cd8f953fad6916R8

Reply:
Same in https://github.com/scality/Arsenal/pull/1855/commits/da9090d3e20de3f102aa3a7f62274384ec80dc24#diff-073d3a5a6a278c5758cbf757c53c2a5e04843f623799fbfd51cd8f953fad6916L105 for the AmzRestore.